File: angular-frontend/src/app/models/video.ts
export class Video {

    constructor(
        private _videoId: number,
        private _videoUrl: string,
        private _videoComment: string,
    ){}

    public get videoId(): number {
        return this._videoId;
    }

    public set videoId(value: number) {
        this._videoId=value;
    }

    public get videoUrl(): string {
        return this._videoUrl;
    }

    public set videoUrl(value: string) {
        this._videoUrl=value;
    }

    public get videoComment(): string {
        return this._videoComment;
    }

    public set videoComment(value: string) {
        this._videoComment=value;
    }
}

File: edge-service/src/main/java/com/ironhack/edgeservice/classes/Video.java
package com.ironhack.edgeservice.classes;

public class Video {

    /** ------------ Properties ------------ **/

    private Integer videoId;
    private String videoUrl;
    private String videoComment;

    /** ------------ Constructors ------------ **/

    public Video() {
    }

    public Video(String videoUrl, String videoComment) {
        setVideoUrl(videoUrl);
        setVideoComment(videoComment);
    }

    /** --------- Getters & Setters --------- **/

    public Integer getVideoId() {
        return videoId;
    }

    public void setVideoId(Integer videoId) {
        this.videoId = videoId;
    }

    public String getVideoUrl() {
        return videoUrl;
    }

    public void setVideoUrl(String videoUrl) {
        this.videoUrl = videoUrl;
    }

    public String getVideoComment() {
        return videoComment;
    }

    public void setVideoComment(String videoComment) {
        this.videoComment = videoComment;
    }
}
